src/components/textarea/styles.ts

```diff
@@ -78,3 +78,13 @@ export const ElTextArea = styled.textarea`
     font-size: var(--font-size-small);
   }
 `
```

Comment:
**note:** New styled component for our "shadow" text area. Importantly, this component is visually hidden in a number of different ways to ensure users will never see it. When I use it, I also hide it from an a11y perspective via `aria-hidden`.